cypress/integration/skills.js

```diff
@@ -0,0 +1,131 @@
+const SkillList = [
+    "Athlethics","Brawl", "Craft", "Drive", "Firearms", "Larceny", "Melee", "Stealth", "Survival",
+    "Animal Ken","Etiquette", "Insight", "Intimidation", "Leadership", "Performance", "Persuasion", "Streetwise", "Subterfuge",
+    "Academics","Awareness", "Finance", "Investigation", "Medicine", "Occult", "Politics", "Science", "Technology"
+]
+const Distributions = {
+    "Balanced": [12,7,5,3,0,0],
+    "Jack-of-all-trades": [8,10,8,1,0,0],
+    "Specialist": [17,3,3,3,1,0]
+}
+const Balanced3LvlSkills = [0,1,2]
+const Balanced2LvlSkills = [3,4,5,6,7]
+const Balanced1LvlSkills = [8,9,10,11,12,13,14]
+const Jack2LvlSkills = [15,16,17]
+const Jack1LvlSkills = [18,19,20]
+const Jack0LvlSkills = [0,2]
+
+function checkNumberOfDots(Item, Expected) {
```

Comment:
mozno by sa tato  funkcia mohla tahat z ineho subory miesto toho aby to bola copy pasta

Reply:
neni to copy-pasta ta funkcia je ina, najmensia bodka v attributes je vzdy init a tu nie